test/e2etest/helper.go

```diff
@@ -80,16 +88,35 @@ func sendPostRequest(t *testing.T, url string, contentType string, body io.Reade
 	defer resp.Body.Close()
 }
 
-func createConfig(t *testing.T, productList []string) TestConfig {
-	var bambooLicense = bambooLicense
-	if len(bambooLicense) == 0 {
-		bambooLicense = os.Getenv("TF_VAR_bamboo_license")
+func getLicense(productList []string, product string) string {
+	license := ""
+	if contains(productList, product) {
+		switch product {
+		case "jira":
+			license = jiraLicense
+		case "confluence":
+			license = confluenceLicense
+		case "bitbucket":
+			license = bitbucketLicense
+		case "bamboo":
+			license = bambooLicense		}
+		if len(license) == 0 {
+			license = os.Getenv(fmt.Sprintf("TF_VAR_%s_license", product))
+		}
 	}
+	return license
+}
+
+func createConfig(t *testing.T, productList []string) TestConfig {
+
 	testConfig := TestConfig{
-		AwsRegion:       GetAvailableRegion(t),
-		BambooLicense:   bambooLicense,
-		EnvironmentName: EnvironmentName(),
-		ResourceOwner:   resourceOwner,
+		AwsRegion:         GetAvailableRegion(t),
+		EnvironmentName:   EnvironmentName(),
+		ResourceOwner:     resourceOwner,
+		JiraLicense:       getLicense(productList, "jira"),
```

Comment:
How does this work if Jira does not support license pre-seeding?

Reply:
This is in test code, and this will not break things. All the logic is setting an environment variable named `<product>_license`.  As far as Jira does not support license pre-seeding, the `helper` function does not produce the environment variable for jira license, or even if it define that variable, jira does not pick it.  
I think even is better if we keep it for just in case if in future jira supports license pre-seeding.

Reply:
If this just doesn't work (I dont see this being supported by Jira anytime soon) then why keep it? Come to think of it, this is even more a reason to get rid of it... Less code to maintain and all that 😄  "Support" for license pre-seeding is also being removed from the Jira Helm Charts. I just think keeping this around is going to make things confusing down the line.

Reply:
fair enough :)

Reply:
Cleaned the code and removed jira license process.